src/meta/meta_ckpt.c

```diff
@@ -697,6 +697,12 @@ __ckpt_load(WT_SESSION_IMPL *session, WT_CONFIG_ITEM *k, WT_CONFIG_ITEM *v, WT_C
         goto format;
     ckpt->write_gen = (uint64_t)a.val;
 
+    /* Runtime write generation isn't needed. */
```

Comment:
I think a comment here would be valuable, but could you update it to be clearer?

Reply:
5c129d6